front/src/utils/date.ts

```diff
@@ -20,7 +20,10 @@ const userTimeZone = dayjs.tz.guess(); // Format : 'Europe/Paris'
  * @param dateTimeString date string in ISO format
  * @returns string "HH:MM:SS"
  */
-export function extractHHMMSS(dateTimeString?: string) {
+export const isoStringToLocalDate = (dateString: string) =>
```

Comment:
Hm, it's not super clear that this function is just formatting the hours+minutes+seconds, and leaving out the day+month+year.

Also other call-sites may need slightly different formatting (e.g. leaving out the seconds), so maybe we should move this inside `ConflictCard` for now?

Reply:
Fixed :)